pkgs/servers/mlflow-server/default.nix

```diff
@@ -10,7 +10,8 @@ py.toPythonApplication
     propagatedBuildInputs = old.propagatedBuildInputs ++ [
       py.boto3
       py.mysqlclient
-    ];
+      py.psycopg2
+    ] ++ additionalPackages;
```

Comment:
This seems unnecessary, since you can just take the resulting python application and call it with `overrideAttrs` to add more `propagatedBuildInputs` without the extra hook, right?

Reply:
I thought so too, but didn't work:
```
  mlflow-server = super.mlflow-server.overrideAttrs (old: rec {
    propagatedBuildInputs = old.propagatedBuildInputs ++ [
      super.python3.packages.psycopg2
    ];
  });
```
I also tried overriding mlflow, but didn't work either.

Reply:
I think it’s a quirk of toPythonApplication, but maybe I made a bonehead mistake

Reply:
you may have to do `overridePythonAttrs`, I'm honestly not sure what the difference is